SampleCode-V5/android-sdk-v5-sample/app-aircraft/src/main/java/dji/sampleV5/aircraft/control/PachKeyManager.kt

```diff
@@ -68,7 +76,20 @@ class PachKeyManager() {
         }
     }
     // Initialize necessary classes
+    private var pachModel: PachWidgetModel = PachWidgetModel.getInstance()
+    var listener: WaypointListener? = null
     val telemService = TuskServiceWebsocket()
+    private var safetyFailures: Array<Int> = arrayOf(0,0,0,0,0)
+    private var action: String = ""
+    private var autonomous: Boolean = false
+    private var endOfFlight: Boolean = false
+    private val safetyWarnings = mapOf(
```

Comment:
Do you think these strings would be better placed in the "resources" folder? it seems like that was the proper approach towards development.

Reply:
Earmarked for changes after the demo day